src/helics/common/JsonProcessingFunctions.cpp

```diff
@@ -16,15 +16,34 @@ SPDX-License-Identifier: BSD-3-Clause
 
 namespace helics::fileops {
 
-bool looksLikeCommandLine(std::string_view testString)
+bool looksLikeJson(std::string_view jsonString)
```

Comment:
```suggestion
bool looksLikeJsonConfig(std::string_view jsonString)
```

Since we're looking for something that specifically looks like a JSON config (string), which is narrower than things that look like JSON.